waffle-provider/package.json

```diff
@@ -35,9 +35,11 @@
     "node": ">=10.0"
   },
   "dependencies": {
+    "@ethereum-waffle/ens": "^3.0.2",
     "ethers": "^5.0.1",
     "ganache-core": "^2.10.2",
-    "@ethereum-waffle/ens": "^3.0.2"
+    "patch-package": "^6.2.2",
+    "postinstall-postinstall": "^2.1.0"
```

Comment:
why this?

Reply:
https://github.com/ds300/patch-package#why-use-postinstall-postinstall-with-yarn